packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -735,7 +735,7 @@ abstract class BoxScrollView extends ScrollView {
 ///  * Using [AutomaticKeepAlive] widgets (inserted by default when
 ///    [addAutomaticKeepAlives] is true). Instead of unconditionally caching the
 ///    child element subtree when scrolling off-screen like [KeepAlive],
-///    [AutomaticKeepAlive] can let whether to cache the subtree be determined
+///    [AutomaticKeepAlive] can let know whether to cache the subtree be determined
```

Comment:
For both of these, I'm not sure this actually makes things better.

AutomaticKeepAlive can let _what_ or _who_ know?

The problem right now is that the sentence is just awkwardly structured. We should just rewrite these two lines to be clearer about responsible parties and what those parties actually do.

Reply:
For example,

```
///  * Using [AutomaticKeepAlive] widgets (inserted by default when
///    [addAutomaticKeepAlives] is true). [AutomaticKeepAlive] allows descendant
///    widgets to control whether the subtree is actually kept alive or not.
///    This behavior is in contrast with [KeepAlive], which will unconditionally keep
///    the subtree alive.
```